tests/e2e/upgrade/wdio.conf.js

```diff
@@ -123,8 +128,13 @@ const servicesStartTimeout = () => {
 // Override specific properties from wdio base config
 const upgradeConfig = Object.assign(wdioBaseConfig.config, {
   specs:
+  // order is important, because we want to upgrade from an older version to current version. validate the upgrade
+  // and then upgrade to master
```

Comment:
Thank you for adding the comment, it helps to understand why the tests are running in that way 🤩